PersonalTrainingDB/3 reports.sql

```diff
@@ -0,0 +1,17 @@
+use PersonalTrainingDB
+go
+
+--I want to see a list of all my current clients presented as follows:
+    --last name, first name, starting weight, current weight, weight lost so far, goal weight if applies,
+    --trainer, how long they have been training with us, ordered by newest clients to oldest clients
+select c.LastName, c.FirstName, c.StartWeight, c.CurrentWeight, WeightLost = (c.StartWeight - c.CurrentWeight), 
+    GoalWeight = case when ClientsGoal like '%Goal Weight%' then ClientsGoal else '' end, c.PrimaryTrainer, YearsSinceBeganTraining = concat(datediff(year, c.SignupDate, getdate()), ' Years')
+from Client c
+order by datediff(month, c.SignupDate, getdate()) desc
```

Comment:
It seems like the order by is incorrect, spec says to show the clients newest to oldest

Reply:
fixed

Reply:
perfect